test/BridgeBaseTest.t.sol

```diff
@@ -48,65 +48,64 @@ contract BridgeBaseTest is Test {
     BridgeLimiter public limiter;
     BridgeTokens public tokens;
 
-/**
-    // TODO: It does not run if it is outside of setUpBridgeTest
-    function testFailInitializeCommitteeAndStakeArraysMustBeOfTheSameLength() public {
-        address[] memory _committee = new address[](5);
-        _committee[0] = committeeMemberA;
-        _committee[1] = committeeMemberB;
-        _committee[2] = committeeMemberC;
-        _committee[3] = committeeMemberD;
-        _committee[4] = committeeMemberE;
-
-        uint16[] memory _stake = new uint16[](4);
-        _stake[0] = 1000;
-        _stake[1] = 1000;
-        _stake[2] = 1000;
-        _stake[3] = 2002;
-
-        vm.expectRevert(bytes("BridgeCommittee: Committee and stake arrays must be of the same length"));
-        committee.initialize(_committee, _stake);
-    }
-
-    // TODO: It does not run if it is outside of setUpBridgeTest
-    function testFailInitializeCommitteeDuplicateCommitteeMember() public {
-        address[] memory _committee = new address[](5);
-        _committee[0] = committeeMemberA;
-        _committee[1] = committeeMemberB;
-        _committee[2] = committeeMemberC;
-        _committee[3] = committeeMemberD;
-        _committee[4] = committeeMemberA;
-
-        uint16[] memory _stake = new uint16[](5);
-        _stake[0] = 1000;
-        _stake[1] = 1000;
-        _stake[2] = 1000;
-        _stake[3] = 2002;
-        _stake[4] = 1000;
-
-        vm.expectRevert(bytes("BridgeCommittee: Duplicate committee member"));
-        committee.initialize(_committee, _stake);
-    }
-
-    // TODO: It does not run if it is outside of setUpBridgeTest
-    function testFailInitializeTotalStakeMustBe10000() public {
-        address[] memory _committee = new address[](4);
-        _committee[0] = committeeMemberA;
-        _committee[1] = committeeMemberB;
-        _committee[2] = committeeMemberC;
-        _committee[3] = committeeMemberD;
-
-        uint16[] memory _stake = new uint16[](4);
-        _stake[0] = 1000;
-        _stake[1] = 1000;
-        _stake[2] = 1000;
-        _stake[3] = 2000;
-
-        vm.expectRevert(bytes("BridgeCommittee: Total stake must be 10000"));
-        committee.initialize(_committee, _stake);
-    }
-*/
-
+    /**
+     * // TODO: It does not run if it is outside of setUpBridgeTest
+     * function testFailInitializeCommitteeAndStakeArraysMustBeOfTheSameLength() public {
+     *     address[] memory _committee = new address[](5);
```

Comment:
what's the plan to fix these?

Reply:
These are the fuzzing tests I was talking about. @georgedigkas should be on this?